src/__tests__/store/reducers/request.reducer.test.js

```diff
@@ -0,0 +1,49 @@
+import requestReducer, { initialState } from '../../../store/reducers/request.reducer';
+import {
+    REQUEST_START,
+    REQUEST_COMPLETE,
+} from '../../../store/constants';
+import getRandomString from '../../../helpers/getRandomString';
+import touchedReducerProps from '../../../helpers/touchedReducerProperties';
+
+describe('request reducer', () => {
+    describe('Default', () => {
+        const result = requestReducer(initialState, {});
```

Comment:
Since there is no other test using `initialState` I can assume that variable is not being used inside the reducer. If that's the case, you can actually use something else as the initial state, so you don't have to export that value in the reducer file.